.gitignore

```diff
@@ -1,8 +1,81 @@
-*.disasm
-*.dll
-*.lib
+### C ###
+# Prerequisites
+*.d
+
+# Object files
 *.o
+*.ko
+*.obj
+*.elf
+
+# Linker output
+*.ilk
+*.map
+*.exp
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Libraries
+*.lib
+*.a
+*.la
+*.lo
+
+# Shared objects (inc. Windows DLLs)
+*.dll
 *.so
-*~
-/build*
+*.so.*
+*.dylib
 
+# Executables
+*.exe
+*.out
+*.app
+*.i*86
+*.x86_64
+*.hex
+
+# Debug files
+*.dSYM/
+*.su
+*.idb
+*.pdb
+
+# Kernel Module Compile Results
+*.mod*
+*.cmd
+.tmp_versions/
+modules.order
+Module.symvers
+Mkfile.old
+dkms.conf
+
+### Meson ###
+# subproject directories
+/subprojects/*
+!/subprojects/packagefiles/
+!/subprojects/*.wrap
+/subprojects/zycore.wrap
+
+# Meson Directories
+meson-logs
+meson-private
+
+# Meson Files
+meson_benchmark_setup.dat
+meson_test_setup.dat
+sanitycheckcpp.cc     # C++ specific
+sanitycheckcpp.exe    # C++ specific
+
+# Ninja
+build.ninja
+.ninja_deps
+.ninja_logs
+
+# Misc
+compile_commands.json
```

Comment:
I recognize this! It's totally wrong. See https://github.com/toptal/gitignore/pull/537

"toptal" is totally unmaintained and IMO also totally junk. Please delete all of it. :)

Reply:
ack

Reply:
done, thanks :)